<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IWidget</class>
 <widget class="QWidget" name="IWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="statesVerticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>States:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="cmbStates">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="btnAddState">
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRemoveState">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnRenameState">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnOrderStateBack">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnOrderStateForward">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRenameState">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename State</string>
   </property>
   <property name="toolTip">
    <string>Rename state</string>
   </property>
  </action>
  <action name="actionAddState">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add State</string>
   </property>
   <property name="toolTip">
    <string>Add a new state for this sprite</string>
   </property>
  </action>
  <action name="actionRemoveState">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remo&amp;ve State</string>
   </property>
   <property name="toolTip">
    <string>Remove the currently selected state</string>
   </property>
  </action>
  <action name="actionOrderStateForwards">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-right.png</normaloff>:/icons16x16/generic-right.png</iconset>
   </property>
   <property name="text">
    <string>Order State For&amp;wards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state forward by 1</string>
   </property>
  </action>
  <action name="actionOrderStateBackwards">
   <property name="icon">
    <iconset resource="../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-left.png</normaloff>:/icons16x16/generic-left.png</iconset>
   </property>
   <property name="text">
    <string>Order State &amp;Backwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state back by 1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../data/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
